src/main/webapp/app/lecture/lecture-unit/lecture-unit-management/create-video-unit/create-video-unit.component.html

```diff
@@ -0,0 +1,9 @@
+<div *ngIf="isLoading" class="d-flex justify-content-center">
+    <div class="spinner-border" role="status">
+        <span class="sr-only">Loading...</span>
```

Comment:
use translations

Reply:
done in 352420623b362c85e24e7e3b6453872b5e60481a